### PR TITLE
release css and fix break .class container

### PR DESCRIPTION
**Pull request by jasonrichdarmawan**
@alfianmrf please review this PR request.

### CODE REVIEW

File: assets/css/style.css [hunk 1]
```diff
@@ -57,6 +57,75 @@
...
+    cursor: pointer;
+    padding: 16px;
+    width: 100%;
+    text-align: left;
+    /* bug border-bottom */
+    border: none;
+    border-bottom: 1px solid #ddd;
+    outline: none;
+    transition: 0.4s;
+}
```
Comment (alfianmrf) [inline assets/css/style.css:185]: please give specific color based on color palette

File: assets/css/style.css [hunk 2]
```diff
@@ -57,6 +57,75 @@
...
+.accordion:after {
+    content: '\02795'; /* Unicode character for "plus" sign (+) */
+    font-size: 13px;
+    color: #777;
+    float: right;
+    margin-left: 5px;
+}
+.active:after {
+    content: "\2796"; /* Unicode character for "minus" sign (-) */
```
Comment (alfianmrf) [inline assets/css/style.css]: you can use icon from font awesome for better customization

File: assets/css/style.css [hunk 3]
```diff
@@ -57,6 +57,75 @@
     color: white;
 }
 
+/* TO DO */
+
+.main2 {
```
Comment (alfianmrf) [inline assets/css/style.css:138]: change font color to #3f3844 like we discuss last night

File: assets/css/style.css (hunk 2 above)
Comment (jasonrichdarmawan) [reply assets/css/style.css]: I use JS to manipulate the <div> with class `.accordion`. Do you know the unicode for this [font-awesome fa-plus](https://fontawesome.com/v4.7.0/icon/plus)

**Review by alfianmrf — APPROVED** [review]

**Pull request merged**